backend/benefit/calculator/models.py

```diff
@@ -426,6 +426,8 @@ def __init__(self, *args, **kwargs):
     amount = models.DecimalField(
         max_digits=7, decimal_places=2, verbose_name=_("row amount")
     )
+    start_date = models.DateField(blank=True, null=True, verbose_name=_("Start date"))
+    end_date = models.DateField(blank=True, null=True, verbose_name=_("End date"))
```

Comment:
+1 for adding start_date and end_date in the CalculationRow. I considered it myself but there wasn't a strong enough use case for it at the time, but for the Ahjo handling it's good to have it here.